packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -586,8 +588,12 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
     }
 
     protected renderResultLineNode(node: SearchInWorkspaceResultLineNode): React.ReactNode {
+        const lineNumber = this.searchInWorkspacePreferences['search.lineNumbers'] ? node.line : '';
         const prefix = node.character > 26 ? '... ' : '';
         return <div className={`resultLine noWrapInfo ${node.selected ? 'selected' : ''}`}>
```

Comment:
Why do you display "..." at the beginning of the string, it does not  align/reduce the string that much and wee have no idea what has been cut !!

Reply:
the ellipsis (...) are not part of this change, and are currently present in master.